content/components/selectpanel.mdx

```diff
@@ -143,30 +110,76 @@ When no results are available, it is essential to offer users an explanation ins
+Inform users about any selection limitations beforehand. Upon reaching the maximum selection, display a warning message, potentially with an upgrade link for paid features. The warning message is labeled with an `aria-live` attribute, ensuring that users with screen readers receive a notification when the message appears.
 
 ### Loading
 
-It is important to provide visual cues to the user when certain processes may take longer than expected. Loading states can be utilized at different stages of using this component to communicate this information effectively.
+Loading states should be used to indicate ongoing processes at various stages, enhancing user awareness and interaction understanding.
 
 <img
   width="1668"
   alt="The image showcases 4 examples of loading states that can appear in the select panel.In all examples, the header displays a title 'Assign people,' a close button, and a search input field. Additionally, each example includes a footer with a cancel and save button. The first example replaces the list with a spinner and a description below it stating 'Fetching people...'. In the second example, the list is replaced by a shimmering skeleton loader, indicating that items will appear there shortly. The third example shows a value in the input field, and the magnifying glass at the beginning of the input field is replaced by a spinner. The list displays previously loaded items. The last example contains no value in the input field, and one item is selected in the list. However, in this case, the apply button is replaced by a spinner enclosed within a rectangle of the same size as the previous apply button."
   src="https://github.com/primer/design/assets/980622/93b6b99c-368b-403c-b604-a1f50920a1ef"
 />
 
-1. **Initial**: Should be used when retrieving the initial data to prevent users from seeing an empty list.
-2. **Initial (skeleton)**: Should be used when retrieving the initial data to prevent users from seeing an empty list.
-3. **Searching/Filtering**: When users perform a search to fetch new items, it is important to maintain their current selections or allow them to continue making selections. In this case, a more minimal loading state should be used to indicate ongoing activity.
-4. **Confirmation**: If your layout does not incorporate optimistic updates or lacks error messaging for failed updates, it is crucial to use a confirmation button as a loading state to clearly communicate the progress of the update.
+1. **Initial**: Used during initial data retrieval to avoid displaying an empty list.
+2. **Initial Skeleton**: Used similarly for initial data retrieval, presenting a skeleton screen instead of an empty list.
+3. **Searching/Filtering**: A minimal loading indicator is used during searches, allowing users to retain or continue their selections.
+4. **Confirmation**: Essential when updates aren't instant or when there's no error messaging for failures, using a confirmation button to indicate update progress.
+
```

Comment:
```suggestion
1. **Initial** is used during initial data retrieval to avoid displaying an empty list.
2. **Initial Skeleton**is used similarly for initial data retrieval, presenting a skeleton screen instead of an empty list.
3. **Searching/Filtering** uses a minimal loading indicator during searches, allowing users to retain or continue their selections.
4. **Confirmation spinner** is essential when updates aren't instant to indicate the progress of the update.

```